xbmc/platform/linux/input/LibInputHandler.h

```diff
@@ -27,6 +28,8 @@ class CLibInputHandler : CThread
 
   void Start();
 
+  bool SetKeymap(std::string layout) { return m_keyboard->SetKeymap(layout); }
```

Comment:
`const std::string&`

Reply:
Move to .cpp file so you don't have to include `LibInputKeyboard.h` in this file. Always avoid pulling in #includes in the header when possible.